.github/workflows/staging.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - "v2"
+    - "test"
```

Comment:
J'me dis qu'utiliser le [workflow_dispatch](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_dispatchinputs) donnerait la latitude de déployer direct depuis l'interface en 1 étape.

Qu'est-ce que t'en pense ?

Reply:
En effet, je modifie en ce sens !